src/Map.js

```diff
@@ -66,6 +66,14 @@ export class Map extends KeyedCollection {
     });
   }
 
+  static ownEntries(obj) {
```

Comment:
typically this naming convention implies that a list of entries will be returned where `obj` is expected to be an instance of a thing that has entries (like `Reflect.ownKeys`) should this maybe be `fromOwnEntries`?